cllc-public-app/Controllers/AdoxioApplicationController.cs

```diff
@@ -284,17 +284,18 @@ public async Task<IActionResult> CreateApplication([FromBody] ViewModels.AdoxioA
             string name = adoxioApplication.AdoxioJobnumber + " Files";
 
             // Create the folder
-            bool folderExists = await _sharePointFileManager.FolderExists(ApplicationDocumentListTitle, folderName);
+            var documentUrlTitle = "adoxio_application";
```

Comment:
I don't agree with this change - it appears we are replacing a constant with a variable which we assign inline.  I'd rather go with a constant.